webvtt.go

```diff
@@ -188,10 +192,17 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 				item.Lines = append(item.Lines, Line{Items: []LineItem{{Text: line}}})
 			default:
 				// This is the ID
-				// TODO Do something with the id
+				// o.Items is empty initially - implementing to save indexe(s) later.
+				if index, err := strconv.Atoi(line); err == nil {
```

Comment:
As for `.srt` files, no need to check for errors here : `index, _ = strconv.Atoi(line)`